TACOCAT.py

```diff
@@ -70,18 +68,28 @@ def test(x,y):
 #Core Temperature Calculations
 #Call axial bulk temperature distribution calculation
 
-FluxPro = np.zeros(Geometry.steps)
-FluxPro[:] = np.cos((np.pi/Hc)*Geometry.z[:])
-
-Tbulk = np.zeros(Geometry.steps) #Initialize Bulk Temperature of Coolant - C
-Tbulk[0] = Tbulkin
+# Bulk Temperature of Coolant in Average Channel - C
+def BulkTemp(FluxPro):

```

Comment:
Add the other inputs to the function bulktemp(FluxPro, geometry, Uinlet, coolant type, etc)